projects/observability/src/shared/components/topology/d3/d3-topology.ts

```diff
@@ -91,6 +95,21 @@ export class D3Topology implements Topology {
     this.hover = new TopologyHover(this.d3Util, this.domRenderer);
     this.click = new TopologyClick(this.d3Util, this.domRenderer);
     this.zoom = new TopologyZoom();
+    this.layout = this.initializeLayout(config.layoutType);
+  }
+
+  private initializeLayout(layoutType?: TopologyLayoutType): TopologyLayout {
+    switch (layoutType) {
+      case TopologyLayoutType.GraphLayout:
```

Comment:
The last one is the currently used layout, that we see in the app-dev. 

And regarding the names,
Tree layout: for graphs without cycle
Graph layout: for graphs with cycle
Forced layout: don't know it's purpose
Custom Tree layout: It's basically tree layout with a different layout
So, aren't they clear already?

Reply:
> So, aren't they clear already?

Appears not! They're all for graphs with cycles, they just use different algorithms to visualize the graph (we only work with graphs, so any that aren't handling graphs or cycles are just bugs). Force layout treats nodes as particles (https://github.com/d3/d3-force), we stopped using it because for whatever reason people felt the hierarchy version was prettier.